api/envoy/extensions/filters/http/header_to_metadata/v4alpha/header_to_metadata.proto

```diff
@@ -90,25 +90,35 @@ message Config {
     option (udpa.annotations.versioning).previous_message_type =
         "envoy.extensions.filters.http.header_to_metadata.v3.Config.Rule";
 
-    // The header that triggers this rule — required.
-    string header = 1
-        [(validate.rules).string = {min_bytes: 1 well_known_regex: HTTP_HEADER_NAME strict: false}];
+    // Specifies that a match will be performed on the value of a header or a cookie.
```

Comment:
More formatter: Delete this, and update the comments for `header` and `cookie` to match the v3 comments.

Reply:
the checker wants me to remove `min_bytes:1` from there as well